data/blog/webview2.md

```diff
@@ -0,0 +1,108 @@
+
+Electron provides APIs for common desktop application needs such as menus, file system access, notifications, and more.
+WebView2 is a component meant to be integrated into an application framework such as WinForms, WPF, or Win32.
+WebView2 does not provide operating system APIs outside the web standard via JavaScript.
+
+Node.js is integrated into Electron.
+Electron applications may use any Node.js API, module, or node-native-addon from the renderer and main processes.
+A WebView2 application does not assume which language or framework the rest of your application is written in.
+There are C++ and C# templates available, but in each case your JavaScript code must proxy any operating system access through the application-host process.
+
+Electron strives to maintain compatibility with the web API, including APIs developed from the [Fugu Project](https://fugu-tracker.web.app/).
+We have a [snapshot of Electron’s Fugu API compatibility](https://docs.google.com/spreadsheets/d/1APQalp8HCa-lXVOqyul369G-wjM2RcojMujgi67YaoE/edit?usp=sharing).
+WebView2 maintains a similar list of [API differences from Edge](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/browser-features).
+
+Electron has a configurable security model for web content, from full-access to full-sandbox.
+WebView2 content is always sandboxed.
+Electron has [comprehensive security documentation](https://www.electronjs.org/docs/tutorial/security) on choosing your security model.
+WebView2 also has [security best practices](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/security).
+
+The Electron source is maintained and available on GitHub.
+Applications can modify can build their own _brands_ of Electron.
+The WebView2 source is not available on GitHub.
+
+Quick Summary:
+
+|                                           | Electron        | WebView2                     |
+| ----------------------------------------- | --------------: | ---------------------------: |
+| Build Dependency                          | Chromium        | Edge                         |
+| Source Available on GitHub                | Yes             | No                           |
+| Shares Edge/Chrome DLLs                   | No              | No                           |
+| Shared Runtime Between Applications       | No              | Optional                     |
+| Application APIs                          | Yes             | No                           |
+| Node.js                                   | Yes             | No                           |
+| Sandbox                                   | Optional        | Always                       |
+| Requires a Separate Application Framework | No              | Yes                          |
```

Comment:
i'd slightly rephrase this - to something like "Requires a Host Application" - there is a "host app" but it isnt necessarily loading another framework

Reply:
Not another web framework, but am I incorrect in saying "Application Framework"? As in, the host app needs a framework. My goal would be for users to understand there's an "app" piece they will need to ship along with webview2.

Reply:
yah - i agree there is a relevant difference here (i often describe it to people that electron is a framework vs webview2 is a control).  so, i agree we need to land the point that one needs a host app, the thing that feels weird is that i wouldn't consider a C++ HWND hosting a webview2 to really be an application framework, and i think there is a "heaviness" (like i'm loading another runtime) that gets implied when saying there is another framework.  does that clarify my comment?

Reply:
oh whoops - i didn't see your slight word tweak before i wrote that. i think your updated wording feels better :)

Reply:
Great! I agree the framework can be built-in or external. Separate could be interpreted as _always external_ which is not true, so I think the new phrasing is better.